blocks/extensions/plot.py

```diff
@@ -136,14 +130,103 @@ def preexec_fn():
             logger.info('Plotting server PID: {}'.format(self.sub.pid))
         else:
             self.sub = None
-        output_server(self.document, url=self.server_url)
+
+        self.session = Session(
+            root_url=self.server_url,
```

Comment:
Where do you see the Travis errors? The errors I see are two: a coverage decreased error and an error from Scrutinizer that seems related to another file: `blocks/bricks/conv.py:238:1: W293 blank line contains whitespace`. Travis build apparently passed.